Source/UniformGradient.swift

```diff
@@ -2,12 +2,19 @@ import UIKit
 
 public struct UniformGradient: Gradient {
 
+    public var angle: GradientAngle?
     public let colors: [UIColor]
+    public let endPoint: CGPoint
```

Comment:
Instead of adding `endPoint`, how about we use that `GradientPoints` type you created for the other gradient, to encapsulate them both?